devdox/tests/unit_test/app/exceptions/test_handlers.py

```
@@ -10,14 +10,20 @@
 
 import pytest
 from fastapi import FastAPI
+from fastapi.exceptions import RequestValidationError
 from starlette.testclient import TestClient
 
 from app.exceptions.custom_exceptions import DevDoxAPIException, UnauthorizedAccess
-from app.exceptions.handlers import (
+from app.exceptions.exception_handlers import (
     devdox_base_exception_handler,
```

Ruff (0.12.2)

`app.exceptions.exception_handlers.devdox_base_exception_handler` imported but unused

Remove unused import

(F401)